lib/libnfs-zdr.c

```diff
@@ -54,6 +54,14 @@ struct zdr_mem {
 
 struct opaque_auth _null_auth;
 
+inline uint32_t libnfs_buf2hl(char *buf) {
+	return ntohl(*(uint32_t *)(void *)buf);
```

Comment:
Could probably change the buf argument to b void *buf and avoid the explicit cast.
But doesn't matter.

I can merge as is once Milhouse verifies it works (or once I set up a rpi emulator)